source/lib/JMSLab/autofill.py

```diff
@@ -32,6 +32,15 @@ def GenerateAutofillMacros(autofill_lists,  autofill_formats, autofill_outfile):
     .tex file
 
     """
+    if type(autofill_lists) != list:
+            raise Exception("Argument 'autofill_lists' must be list")
+            
+    nested_list = any(isinstance(i, list) for i in autofill_lists)
+    
+    if (nested_list == True and type(autofill_formats) == str) or (nested_list == False 
+                                                                   and type(autofill_formats) == list):
```

Comment:
You can replace `nested_list == True` with just `nested_list` and `nested_list == False` with `not nested_list`. You can also replace the `==` in `type(autofill_formats) == str` and `type(autofill_formats) == list` with `is` and you end up with a sentence you can almost read :)

Also, it may look a bit nicer to break the line on the `or`?

Reply:
Thanks @santiagohermo for the pointers! Addressed in [8915073](https://github.com/JMSLab/Template/pull/57/commits/8915073e00ed687c39568d2f2f5a058c9eed1a26).